.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/Passes/vendors/googletest"]
```

Comment:
Should this be shallow?

Reply:
For me it is fine - I prefer to manually initialise submodules. However, ultimately, this is up to you.